src/client/components/share-button/share-button.vue

```diff
@@ -0,0 +1,286 @@
+<template>
+  <div class="social-share">
+    <button class="social-share__button" @click="showSharingOptions"></button>
+
+    <div class="social-share__links">
+      <a :href="`https://twitter.com/home?status=${url}`"
```

Comment:
You can rename this class name to `social-share__link`, which is easier to read.